pkg/apis/servicecatalog/validation/binding.go

```diff
@@ -219,9 +220,15 @@ func validateServiceBindingUpdate(binding *sc.ServiceBinding) field.ErrorList {
 // to the spec to go through.
 func internalValidateServiceBindingUpdateAllowed(new *sc.ServiceBinding, old *sc.ServiceBinding) field.ErrorList {
 	errors := field.ErrorList{}
-	if old.Generation != new.Generation && old.Status.ReconciledGeneration != old.Generation {
-		errors = append(errors, field.Forbidden(field.NewPath("spec"), "another change to the spec is in progress"))
+
+	// If the OriginatingIdentityLocking feature is set then don't allow spec updates
+	// if processing of the current generation hasn't finished yet
+	if utilfeature.DefaultFeatureGate.Enabled(scfeatures.OriginatingIdentityLocking) {
+		if old.Generation != new.Generation && old.Status.ReconciledGeneration != old.Generation {
```

Comment:
Can we get this changed so that it allow the request go go thru when its the same user?  And the other check below too?

Reply:
I don't want to change the behavior in this PR, your change should be a follow-up to this one.